### PR TITLE
rebuild

### PR DESCRIPTION
**Pull request by ocefpaf**
Hopefully fixes https://github.com/conda-forge/fiona-feedstock/issues/86

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by ocefpaf** [comment]
@isuruf this does not like the new syntax that brings the toolchain stuff :-(
Another on that used to work just fine before this change and it is broken now.

**Comment by ocefpaf** [comment]
It is pulling `libgdal: 2.3.1-ha6e4b24_0`! I really do not get the new syntax at all!!

**Comment by ocefpaf** [comment]
We are still getting `libspatialite: 4.3.0a-h72746d6_18 defaults` in Python 3.5. Argh!!!

File: recipe/build.sh [hunk 1]
```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+export CPPFLAGS="$CPPFLAGS -I$PREFIX/include"
+
+# Filter out -std=.* from CXXFLAGS as it disrupts checks for C++ language levels.
```
Comment (isuruf) [inline recipe/build.sh]: Do you know which package sets `-std=.*`. It's not toolchain AFAIK.

Comment (ocefpaf) [reply recipe/build.sh]: I got that from defaults but already removed it. It did not help with the issue here.

**Comment by isuruf** [comment]
Issue is with libspatialite which needs to be rebuilt with updated pinnings

File: recipe/meta.yaml [hunk 2]
```diff
@@ -8,41 +8,32 @@ source:
...
     - m2w64-xz  # [win]
-    - setuptools
-    - cmake  # [win]
-    - numpy 1.11.*
-    - libgdal {{ version }}
-    - vc 9  # [win and py27]
-    - vc 14  # [win and (py35 and py36)]
+    - pip
+    - numpy
+    - {{ pin_compatible('libgdal', exact=True) }}
```
Comment (isuruf) [inline recipe/meta.yaml]: Change this to just `libgdal` and keep the `pin_compatible` for `run`

Comment (ocefpaf) [reply recipe/meta.yaml]: I already did "manually" but left the jinja there afterwards. Still got `libspatialite` from defaults though.

**Comment by isuruf** [comment]
Btw, you'll have to build python 2.7 with new pinnings to get the latest libspatialite.

**Comment by ocefpaf** [comment]
> Issue is with libspatialite which needs to be rebuilt with updated pinnings

I did that some time ago and just re-did it to double check.

> Btw, you'll have to build python 2.7 with new pinnings to get the latest libspatialite.

I'm working on that but Python 3.6.6 (and fixing https://github.com/conda-forge/libnetcdf-feedstock/pull/49) are my priorities at the moment.

**Pull request merged**